cost/azure/unattached_volumes/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v4.1
+
+- bugfix for logic reguarding activity log and what constitutes as unattached/unused. Changed logic to verify that activiies lenght is 0 rather than using "toBeDetached" keyword
```

Comment:
typo:
```suggestion
- bugfix for logic regarding activity log and what constitutes unattached/unused. Changed logic to verify that activities length is 0 rather than using "toBeDetached" keyword
```